src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.NativeAOT.targets

```diff
@@ -0,0 +1,74 @@
+    <!-- Example settings from: https://github.com/jonathanpeppers/Android-NativeAOT/blob/ea69d122cdc7de67aa6a5db14b7e560763c63cdd/DotNet/libdotnet.targets -->
+    <PropertyGroup>
+      <_NdkSysrootAbi>aarch64-linux-android</_NdkSysrootAbi>
+      <_NdkClangPrefix>aarch64-linux-android21-</_NdkClangPrefix>
+      <_NdkPrebuiltAbi Condition=" $([MSBuild]::IsOSPlatform('osx')) ">darwin-x86_64</_NdkPrebuiltAbi>
+      <_NdkPrebuiltAbi Condition=" $([MSBuild]::IsOSPlatform('linux')) ">linux-x86_64</_NdkPrebuiltAbi>
+      <_NdkSysrootDir>$(_AndroidNdkDirectory)toolchains/llvm/prebuilt/$(_NdkPrebuiltAbi)/sysroot/usr/lib/$(_NdkSysrootAbi)</_NdkSysrootDir>
+      <_NdkBinDir>$(_AndroidNdkDirectory)toolchains/llvm/prebuilt/$(_NdkPrebuiltAbi)/bin/</_NdkBinDir>
+      <CppCompilerAndLinker>$(_NdkBinDir)$(_NdkClangPrefix)clang++</CppCompilerAndLinker>
+      <ObjCopyName>$(_NdkBinDir)llvm-objcopy</ObjCopyName>
+    </PropertyGroup>
+    <ItemGroup>
+      <LinkerArg Include="-Wl,--undefined-version" />
+    </ItemGroup>
+  </Target>
+
+  <!-- https://github.com/xamarin/xamarin-macios/blob/c43d4ea40bc777969e3b158cf46446df292d8449/dotnet/targets/Xamarin.Shared.Sdk.targets#L541-L550 -->
+  <Target Name="_AndroidBeforeIlcCompile">
+    <PropertyGroup>
+      <RunILLink>true</RunILLink>
+      <!--
+        We want to suppress warnings from trimmer and only show warnings from ILC.
+        Otherwise, you would get 2x for every warning.
+      -->
+      <_OriginalSuppressTrimAnalysisWarnings>$(SuppressTrimAnalysisWarnings)</_OriginalSuppressTrimAnalysisWarnings>
+      <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="_AndroidComputeIlcCompileInputs">
+    <PropertyGroup>
+      <!-- Turn trimmer warnings back to original value -->
+      <SuppressTrimAnalysisWarnings>$(_OriginalSuppressTrimAnalysisWarnings)</SuppressTrimAnalysisWarnings>
+      <!-- HACK: -targetos:android results in error, so use linux -->
+      <_targetOS>linux</_targetOS>
+    </PropertyGroup>
+    <ItemGroup>
+      <!-- Give ILLink's output to ILC -->
+      <IlcCompileInput Remove="@(IlcCompileInput)" />
+      <IlcCompileInput Include="$(IntermediateLinkDir)$(TargetName)$(TargetExt)" />
+      <IlcReference Remove="@(IlcReference)" />
+      <IlcReference Include="@(ManagedAssemblyToLink->'$(IntermediateLinkDir)%(Filename)%(Extension)')" Condition="Exists('$(IntermediateLinkDir)%(Filename)%(Extension)')" Exclude="@(IlcCompileInput)" />
+      <!-- HACK: these are missing from microsoft.netcore.app.runtime.linux-bionic-arm64 -->
+      <AutoInitializedAssemblies Remove="System.Private.StackTraceMetadata" />
+      <AutoInitializedAssemblies Remove="System.Private.TypeLoader" />
+      <AutoInitializedAssemblies Remove="System.Private.Reflection.Execution" />
+      <AutoInitializedAssemblies Remove="System.Private.Reflection.DisabledReflection" />
```

Comment:
This got past the previous error, but this just showcases the current blocker:
```
Expected type 'Internal.Runtime.CompilerHelpers.ThrowHelpers' not found in module 'System.Private.CoreLib'
```

We are missing files in:
```
% ls ~/.nuget/packages/microsoft.netcore.app.runtime.linux-bionic-arm64/9.0.0/runtimes/linux-bionic-arm64/lib/net9.0/System.Private.*.dll
~/.nuget/packages/microsoft.netcore.app.runtime.linux-bionic-arm64/9.0.0/runtimes/linux-bionic-arm64/lib/net9.0/System.Private.DataContractSerialization.dll
~/.nuget/packages/microsoft.netcore.app.runtime.linux-bionic-arm64/9.0.0/runtimes/linux-bionic-arm64/lib/net9.0/System.Private.Uri.dll
~/.nuget/packages/microsoft.netcore.app.runtime.linux-bionic-arm64/9.0.0/runtimes/linux-bionic-arm64/lib/net9.0/System.Private.Xml.Linq.dll
~/.nuget/packages/microsoft.netcore.app.runtime.linux-bionic-arm64/9.0.0/runtimes/linux-bionic-arm64/lib/net9.0/System.Private.Xml.dll
```